.github/workflows/allthekeeps-testnet.yml

```
@@ -0,0 +1,103 @@
+name: AllTheKeeps / Testnet
```

Can we have one workflow for any network `.github/workflows/allthekeeps.yml`?
```suggestion
name: AllTheKeeps
```

---

This particular workflow handles specifically Ropsten network. All the scripts that are triggered and dApp is pointing to Ropsten. ATK on mainnet is in hands of community.